backend/src/main.ts

```diff
@@ -15,7 +15,7 @@ const io = new Server(httpServer, {
 io.on("connection", (socket) => {
     console.log(`Got new connection: ${socket.id}`);
     socket.on("coffee", (data) => {
-        io.sockets.emit("coffee", data);
+        io.sockets.emit("coffee", JSON.stringify({...JSON.parse(data), broadcastAt: new Date().toISOString()}));
```

Comment:
Wrap it like it's ~hot~readable 🥺

```suggestion
        io.sockets.emit("coffee", JSON.stringify({
            ...JSON.parse(data),
            broadcastAt: new Date().toISOString()
        }));
```